crates/sui-verifier/src/id_leak_verifier.rs

```diff
@@ -59,9 +63,7 @@ fn verify_id_leak(module: &CompiledModule) -> Result<(), ExecutionError> {
             FunctionView::function(module, FunctionDefinitionIndex(index as u16), code, handle);
         let initial_state = AbstractState::new(&func_view);
         let mut verifier = IDLeakAnalysis::new(&binary_view, &func_view);
-        verifier
-            .analyze_function(initial_state, &func_view)
-            .map_err(|e| e.finish(Location::Module(module.self_id())))?;
+        verifier.analyze_function(initial_state, &func_view)?;
```

Comment:
This highlights part of the issue: `e` is now `ExecutionError`, and there doesn't appear to be a convenient way to map `e` to something reasonable that reports, e.g., `ModuleId`. I thought about [changing or minting a new error](https://github.com/MystenLabs/sui/blob/719304331c8f0ab3f3f95af26855a02cf14cc1f0/crates/sui-types/src/messages.rs#L1466), maybe something like `SuiMoveVerificationError(Option<ModuleId>)`.